### DISCUSSION TITLE
How to deal with login screen?

### DISCUSSION THREAD

**Discussion by SapitoSucio** [RxCP/rxcp#258] [Q&A]
Basically title

**Comment by johngerome** [comment]
Hi @SapitoSucio, 

Are looking for the admin's login page and the dashboard? it's still in progress and will be released in the beta version soon.
If you're familiar with how API works you can actually create your own login screen for your **website** (no need for the admin side cause this will be built in), you can check the API documentation [here](https://documenter.getpostman.com/view/390924/VUjSGPdG).
Btw the instructions on how to install this CP including the demo will also be in the beta version.
